index.html

```diff
@@ -0,0 +1,209 @@
+            <li><img src="images/javaScript.png" alt=""/></li>
+          </ul>
+          <a href="#"><img class="see-proj" src="images/see.png" alt=""/></a>
+        </div>
+      </div>
+      <div class="project2">
+        <img src="images/portfolio1.png" alt=""/>
+        <div class="proj-desc">
+          <h2>Multi-post <br>Story</h2>
+          <ul class="list1">
+            <li>CANOPY</li>
+            <li><img src="images/counter.png" alt=""/></li>
+            <li>Back End Dev</li>
+            <li><img src="images/counter.png" alt=""/></li>
+            <li>End</li>
+          </ul>
+          <p class="parag">
+            A daily selection of privately personalized reads; no accounts
+            orsign-ups required.
+          </p>
+          <ul class="proj-tech">
+            <li><img src="images/htmli.png" alt="" /></li>
+            <li><img src="images/css.png" alt=""/></li>
+            <li><img src="images/javaScript.png" alt=""/></li>
+          </ul>
+          <img class="see-proj" src="images/see.png" alt="" />
+        </div>
+      </div>
+      <div class="project3">
+        <img src="images/portfolio.png" alt="" />
+        <div class="proj-desc">
+          <h2>Tonic</h2>
+          <ul class="list1">
+            <li>CANOPY</li>
+            <li><img src="images/counter.png" alt="" /></li>
+            <li>Back End Dev</li>
+            <li><img src="images/counter.png" alt=""/></li>
+            <li>End</li>
+          </ul>
+          <p class="parag">
+            A daily selection of privately personalized reads; no accounts
+            orsign-ups required.
+          </p>
+          <ul class="proj-tech">
+            <li><img src="images/htmli.png" alt=""/></li>
+            <li><img src="images/css.png" alt="" /></li>
+            <li><img src="images/javaScript.png" alt=""/></li>
+          </ul>
+          <img class="see-proj" src="images/see.png" alt=""/>
+        </div>
+      </div>
+      <div class="project4">
+        <img src="images/multi.png" alt="" />
+        <div class="proj-desc">
+          <h2>Multi-post <br>Story</h2>
+          <ul class="list1">
+            <li>CANOPY</li>
+            <li><img src="images/counter.png" alt=""/></li>
+            <li>Back End Dev</li>
+            <li><img src="images/counter.png" alt="" /></li>
+            <li>End</li>
+          </ul>
+          <p class="parag">
+            A daily selection of privately personalized reads; no accounts
+            orsign-ups required.
+          </p>
+          <ul class="proj-tech">
+            <li><img src="images/htmli.png" alt=""/></li>
+            <li><img src="images/css.png" alt=""/></li>
+            <li><img src="images/javaScript.png" alt="" /></li>
+          </ul>
+          <img class="see-proj" src="images/see.png" alt=""/>
+        </div>
+      </div>
+    </section>
+    <section class="about-sect">
+      <div class="about">
+        <h2 class="about-intro intro">About <br />Myself</h2>
```

Comment:
- [ ] Kindly select in you css file the h2 with through his `class` and ad this code `font-size: xx-large;` it gonna increase the size of the title.